src/hooks/custom-options.js

```diff
@@ -16,4 +16,9 @@ exports.customOptionsHook = function (raw, options) {
         options.context.useHeaderGenerator = raw.useHeaderGenerator;
         delete raw.useHeaderGenerator;
     }
+
+    if ('insecureParser' in raw) {
+        options.context.insecureParser = raw.insecureParser;
+        delete raw.insecureParser;
+    }
```

Comment:
Is there a reason not to use insecureHTTPParser as the name?

Reply:
`insecureParser` is shorter but I guess I should've kept `insecureHTTPParser` for comapt?